ansible/tasks/health_check.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Wait epoch node API to boot
+  wait_for:
+    port: "{{ config.http.external.port }}"
+    host: "{{ ansible_ssh_host }}"
+    timeout: 30
```

Comment:
It is ok - though just for ensuring we are on same page I notice in normal deployment it is more https://github.com/aeternity/epoch/blob/ac7f53b18d612a35ed8575544e162cb4cdbbc68f/deployment/ansible/deploy.yml#L155

Reply:
I tested the timeout and and really prefer "fail fast", 30 seconds should be enough to boot up. 5 minutes is away too much.

I know it's different on epoch repo deploy.yml, I'll probably change it there in future too.

P.S. This is one of the reasons I didn't initially added health checks - it's a duplicate of epoch repo and I didn't come up with good solution to this yet.